WordPress/Classes/ViewRelated/Post/Preview/PreviewWebKitViewController.swift

```diff
@@ -70,11 +70,19 @@ class PreviewWebKitViewController: WebKitViewController {
         super.init(configuration: configuration)
     }
 
+    @objc override init(configuration: WebViewControllerConfiguration) {
```

Comment:
Since the new constructor adds one more use case we can update the `PreviewWebKitViewControllerTests` and add a test case for the site preview validating the buttons that are expected to be present.

Reply:
Good idea. I've added a test for this case as well.